packages/toolkit/src/view/settings/api-tokens/SelectOrganization.tsx

```diff
@@ -0,0 +1,127 @@
+"use client";
+
+import * as React from "react";
+import Link from "next/link";
+import { Nullable, UserMembership } from "instill-sdk";
+
+import { Icons, Select } from "@instill-ai/design-system";
+
+import { CodeBlock } from "../../../components";
+import { defaultCodeSnippetStyles } from "../../../constant";
+import {
+  InstillStore,
+  useAuthenticatedUser,
+  useInstillStore,
+  useShallow,
+  useUserMemberships,
+} from "../../../lib";
+
+const selector = (store: InstillStore) => ({
+  accessToken: store.accessToken,
+  enabledQuery: store.enabledQuery,
+});
+
+export const SelectOrganization = () => {
+  const { accessToken, enabledQuery } = useInstillStore(useShallow(selector));
+
+  const me = useAuthenticatedUser({
+    enabled: enabledQuery,
+    accessToken,
+  });
+
+  const organizations = useUserMemberships({
+    enabled: enabledQuery && me.isSuccess,
+    userID: me.isSuccess ? me.data.id : null,
+    accessToken,
+  });
+
+  const [selectedOrg, setSelectedOrg] = React.useState<
+    Nullable<string> | undefined
+  >(null);
+
+  React.useEffect(() => {
+    if (organizations.isSuccess && organizations.data) {
+      setSelectedOrg(organizations?.data[0]?.organization?.id);
+    }
+  }, [organizations.isSuccess, organizations.data]);
+
+  React.useEffect(() => {
+    console.log("Selected organization:", selectedOrg);
+  }, [selectedOrg]);
+
+  const getSelectedOrgUid = React.useCallback(
+    (orgId: Nullable<string> | undefined): string => {
+      if (!organizations.isSuccess) return "Loading...";
+      const org = organizations.data.find(
+        (membership) => membership.organization.id === orgId,
+      );
+      return org ? org.organization.uid : "Loading...";
+    },
+    [organizations.isSuccess, organizations.data],
+  );
+
+  const codeString = React.useMemo(() => {
+    return `--header Instill-Requester-Uid:${getSelectedOrgUid(selectedOrg)}`;
+  }, [selectedOrg, getSelectedOrgUid]);
+
+  if (organizations.isSuccess && organizations.data.length === 0) {
+    return null;
+  }
+
+  return (
+    <div className="flex gap-4 p-4 rounded border border-semantic-bg-line">
+      <Icons.AlertCircle className="w-6 h-6 stroke-semantic-node-connector-on" />
+      <div className="flex flex-col flex-1 gap-3">
+        <p className="text-semantic-fg-secondary product-body-text-3-regular">
+          API tokens are tied to individual users, not organizations. To charge
+          an organization for an API request, include the{" "}
+          <span className="product-body-text-3-medium">
+            Instill-Requester-Uid
+          </span>{" "}
+          header. If you omit this header, personal credits will be used. For
+          more details, refer to the{" "}
+          <Link
+            href="/docs/api-tokens"
+            className="text-semantic-accent-default"
+          >
+            documentation
+          </Link>
```

Comment:
@thewbuk This URL is wrong, we need to provide absolute path for navigation between different domain

`https://www.instill.tech/docs/core/token`